db/migrations/20221005070101_FAQs.sql

```diff
@@ -0,0 +1,10 @@
+-- migrate:up
+CREATE TABLE FAQs(
```

Comment:
```suggestion
CREATE TABLE frequently_asked_questions(
```
컨벤션에 맞게 작성해 주세요~ 그리고 줄임말보다는 풀어쓰는 것이 명확할 것 같습니다.